tsconfig.json

```diff
@@ -4,7 +4,9 @@
 		"target": "ES2020",
 		"declaration": true,
 		"outDir": "./out",
-		"lib": ["es2020", "dom"]
+		"lib": ["es2020"],
+		"types": ["node"],
+		"moduleResolution": "Node"
```

Comment:
removed `dom` from `lib` (because we are not running in browser) and set moduleResolution to `Node`